.github/actions/linux/build-libmongocrypt/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Build libmongocrypt"
+description: "Installs libmongocrypt"
+inputs:
+  version:
+    description: "Libmongocrypt version to install (major.minor)"
+    required: true
+runs:
+  using: composite
+
+  steps:
+    - name: Add repository key
+      shell: bash
+      run: sudo sh -c 'curl -s --location https://pgp.mongodb.com/libmongocrypt.asc | gpg --dearmor >/etc/apt/trusted.gpg.d/libmongocrypt.gpg'
+
+    - name: Add repository
+      shell: bash
+      working-directory: /tmp
+      # Note: no packages for Ubuntu 24.04 noble exist, so we use those for 22.04
```

Comment:
Are there any plans to have one later? If not, it will get complicated when the upgrade to Ubuntu 24.04 is enforced by GitHub.

Reply:
I created [MONGOCRYPT-813](https://jira.mongodb.org/browse/MONGOCRYPT-813) for this. Note that I'm already using 24.04 and the PPA works for the time being. Once the ticket is resolved, I'd update this accordingly.